geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/remote/CommandExecutor.java

```diff
@@ -207,4 +219,32 @@ protected Object invokeCommand(Object command, GfshParseResult parseResult) {
 
     return resultModel;
   }
+
+  @VisibleForTesting
+  boolean lockCMS(Object command) {
+    if (cmsDlockService == null) {
+      return false;
+    }
+    if (!(command instanceof GfshCommand)) {
```

Comment:
How about combining all of the if into an AND condition.

if (cmsDlockService == null && !(command instanceof GfshCommand) &&...)

Reply:
I think this reads better and I plan to add comments in each if block